### PR TITLE
[MST-665] Add missing 'rejected' status to onboarding attempt statuses

### PR DESCRIPTION
**Pull request by bseverino**
**JIRA:**

[MST-665](https://openedx.atlassian.net/browse/MST-665)

**Pre-Merge Checklist:**

- [x] Described your changes in `CHANGELOG.rst`
- [x] Confirmed Github reports all automated tests/checks are passing.
- [x] Approved by at least one additional reviewer.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ Change Log
 
 Unreleased
 ~~~~~~~~~~
+* Add missing `rejected` status to list of onboarding attempt statuses.
```
Comment (schenedx) [inline CHANGELOG.rst:16]: Let's release it with a minor version.

File: edx_proctoring/tests/test_views.py [hunk 2]
```diff
@@ -1118,6 +1118,7 @@ def test_multiple_status_filters(self):
         (ProctoredExamStudentAttemptStatus.ready_to_submit,
             InstructorDashboardOnboardingAttemptStatus.proctoring_started),
         (ProctoredExamStudentAttemptStatus.submitted, InstructorDashboardOnboardingAttemptStatus.submitted),
+        (ProctoredExamStudentAttemptStatus.rejected, InstructorDashboardOnboardingAttemptStatus.rejected),
```
Comment (schenedx) [inline edx_proctoring/tests/test_views.py:1121]: Looks like you have a failed test. Please fix.

**Review by schenedx — COMMENTED** [review]
See inline comments

File: edx_proctoring/tests/test_views.py (hunk 2 above)
Comment (bseverino) [reply edx_proctoring/tests/test_views.py:1122]: This test isn't failing locally for some reason. Going to see if I can figure out why.

Comment (bseverino) [reply edx_proctoring/tests/test_views.py:1129]: I think `MockGradesService` and `MockCertificateService` were getting torn down inbetween test suites, so added those fixed it.

**Review by MichaelRoytman — APPROVED** [review]

**Pull request merged**